### ISSUE TITLE
Wooden Axe not useable

### ISSUE THREAD

**Issue by Deregorn** [breakinblocks/nutsandboltstorqued#159]
The wooden axe is not usable. I can't check right now, but I'd assume there's WorldEdit in the pack, which does this.

**Comment by Saereth** [comment]
yeah its the worldedit wand will disable by default for next release

**Comment by Baughn** [comment]
Having just started a new server, it appears to be disabled by default. I assume this bug can be closed.

**Comment by Saereth** [comment]
It's a singleplayer bug only but its fixed in the next update so I was just leaving this open until the update is out.
